cpmpy/solvers/minizinc.py

```diff
@@ -135,6 +142,8 @@ def __init__(self, cpm_model=None, subsolver=None):
         """
         if not self.installed():
             raise Exception("CPM_minizinc: Install the python package 'minizinc'")
+        if not self.executable_installed():
```

Comment:
This does not change the functionality, but maybe change this to an `elif` or change the `elif` below to a `if` for consistency.